packages/vscode-extension/src/service/TranslationService.ts

```diff
@@ -0,0 +1,31 @@
+/*
+** Copyright (c) 2019 Oracle and/or its affiliates.  All rights reserved.
+** Licensed under the Universal Permissive License v 1.0 as shown at https://oss.oracle.com/licenses/upl.
+*/
+const ApplicationConstants = require('../../../node-cli/src/ApplicationConstants');
+const FileUtils = require('../../../node-cli/src/utils/FileUtils');
```

Comment:
reference node-cli by doing `require('@oracle/suitecloud-cli/...` better, to not mess up with module dependencies